spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,23 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    let(:test_ot) { FactoryGirl.create(:orchestration_template_cfn_with_content) }
+
+    it 'copy template with the same content' do
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      run_post(orchestration_templates_url(test_ot.id), gen_request(:copy))
+      expect(response.status).to eq(400)
```

Comment:
just a suggestion, but this could also be written
`expect(response).to have_http_status(:forbidden)` which is nice for readability